src/filter/median.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Created by Cristian on 18/07/2015.
+ */
+//'use strict'
+
+import Image from '../image';
+//k: size of kernel (k*k)
+export default function medianFilter(k){
+    this.checkProcessable({
```

Comment:
checkProcessable requires as first parameter the name of the procedure. In this case 'medianFilter'